programs/uxd/src/state/controller.rs

```diff
@@ -81,10 +84,15 @@ pub struct Controller {
     // Total amount of profits collected into the treasury by any depository
     pub profits_total_collected: u128,
 
-    // The configured depository balancing weights
+    // The configured router depositories balancing weights
     pub identity_depository_weight_bps: u16,
-    pub mercurial_vault_depository_0_weight_bps: u16,
```

Comment:
Ok this keeps the same memory layout so all good!

Reply:
Yes ser, also this part hasnt been deployed yet, so still time to change if we want?

Reply:
No it's cool, most important is to not wreck the memory layout, we are like on a gameboy here so better not loose the offset of each property. 
Eventually we want to refactor and remove the registered depositories but I think that's another beast

Reply:
Ok sounds good 👌